### PR TITLE
[feat] Update Headers 

### PR DESCRIPTION
**Pull request by amyjchen**
### Changes Made:
- updates headers on read pages
- makes tweaks to text alignment on profile page

| Before | After | After (narrow screen) |
| --- | --- | --- |
|Screenshot 2024-10-31 at 14 52 08|Screenshot 2024-10-31 at 14 51 36| no change |
|Screenshot 2024-10-31 at 14 53 21|Screenshot 2024-10-31 at 14 52 46|Screenshot 2024-10-31 at 14 56 31|
|Screenshot 2024-10-31 at 14 53 26|Screenshot 2024-10-31 at 14 52 54|Screenshot 2024-10-31 at 14 56 39|
|Screenshot 2024-10-31 at 14 53 29|Screenshot 2024-10-31 at 14 53 00|Screenshot 2024-10-31 at 14 56 45|
|Screenshot 2024-10-31 at 14 53 35|Screenshot 2024-10-31 at 14 53 09| no change |
|Screenshot 2024-10-31 at 15 06 49 | Screenshot 2024-10-31 at 15 06 34| no change|

### CODE REVIEW

**Comment by somethingnew2-0** [comment]
image

Can the group icon on the right be vertically centered?

**Comment by amyjchen** [comment]
> image
> 
> Can the group icon on the right be vertically centered?

yup, just updated! 

Screenshot 2024-11-01 at 10 16 56

File: src/pages/apps/Read.tsx [hunk 1]
```diff
@@ -76,51 +76,48 @@ export default function ReadApp() {
...
+                            marginTop: '5px',
+                            bgcolor: (theme) =>
+                              tagMap.active_tag!.enabled ? 'primary' : theme.palette.action.disabled,
+                          }}
+                        />
+                      ))}
+                    </Box>
+                  ) : null}
+                </Stack>
+                <Divider />
```
Comment (somethingnew2-0) [inline src/pages/apps/Read.tsx:108]: When the current user isn't an Access Admin (member of App-Access-Owners) or isn't an App Owner (owner of App Owners group), they don't see anything below this divider. Is there some way we could remove it for that case?

Screenshot 2024-11-01 at 1 42 25 PM

Comment (amyjchen) [reply src/pages/apps/Read.tsx:108]: Ahhh good point. I'll update there and in tags

Comment (amyjchen) [reply src/pages/apps/Read.tsx]: updated!

**Review by somethingnew2-0 — APPROVED** [review]

**Pull request merged**